selftest/tc/main.go

```diff
@@ -56,12 +56,40 @@ func main() {
 
 	var tcOpts bpf.TcOpts
 	tcOpts.ProgFd = int(tcProg.GetFd())
+	tcOpts.Flags = bpf.BpfTcFReplace
+	tcOpts.Handle = 1
 	err = hook.Attach(&tcOpts)
 	if err != nil {
 		fmt.Fprintln(os.Stderr, err)
 		os.Exit(-1)
 	}
 
+	// test for query
+	tcOpts.Flags = 0
+	tcOpts.ProgFd = 0
+	tcOpts.ProgId = 0
```

Comment:
For Query only progfd and progid. https://elixir.bootlin.com/linux/v6.8.4/source/tools/testing/selftests/bpf/prog_tests/tc_bpf.c#L52